dev/SapphireTest.php

```diff
@@ -43,11 +43,6 @@ class SapphireTest extends PHPUnit_Framework_TestCase {
 	 */
 	protected $usesDatabase = null;
 
-	/**
-	 * @deprecated since version 4.0
-	 */
-	protected $originalMailer;
```

Comment:
Unfortunately we cannot remove $originalMailer until 4.0, as semver disallows removal of API in patch releases.